binaries.json

```diff
@@ -0,0 +1,36 @@
+{
+    "released": [
+        "tezos-client",
+        "tezos-admin-client",
+        "tezos-node",
+        "tezos-signer",
+        "tezos-codec",
+        "tezos-baker-PtMumbai",
+        "tezos-accuser-PtMumbai",
+        "tezos-smart-rollup-client-PtMumbai",
+        "tezos-smart-rollup-node-PtMumbai",
+        "tezos-baker-PtNairob",
+        "tezos-accuser-PtNairob",
+        "tezos-smart-rollup-client-PtNairob",
+        "tezos-smart-rollup-node-PtNairob"
+    ],
+    "candidates": [
+        "tezos-smart-rollup-client-Proxford",
+        "tezos-smart-rollup-client-PtNairob",
+        "tezos-smart-rollup-node-Proxford",
+        "tezos-smart-rollup-node-PtNairob",
+        "tezos-baking",
+        "tezos-sapling-params",
```

Comment:
They both should be present in `released` list too. And they both shouldn't be checked with `--version` command, only the fact that they installed successfully.